graphql/schema.js

```diff
@@ -0,0 +1,34 @@
+const typeDefs = `#graphql
+
+  type Book {
+    title: String
+    author: String
```

Comment:
should author be a string or a Node? 

When should we use node vs type?

Reply:
This schema was just for test and was never used actually. I am sorry I forgot to delete it